docs/gadgets/traceloop.md

```diff
@@ -8,7 +8,7 @@ some differences:
 
 * traceloop uses eBPF instead of ptrace
 * traceloop&#39;s tracing granularity is the container instead of a process
-* traceloop&#39;s traces are recorded in a fast, in-memory, overwritable ring
+* traceloop&#39;s traces are recorded in a fast, in-memory, over writable ring
```

Comment:
I'm not a native speaker but overwritable seems better than "over writable". There are a lot of hits for it on Google so I think it's fine -> https://www.google.com/search?client=firefox-b-lm&q=overwritable.